share/spice/libraries/triggers.txt

```diff
@@ -0,0 +1,11 @@
+go
```

Comment:
The libraries.io API doesn't support "go get" queries.  I think more than two words plus original word, as in, "wordpress woo commerce" would get same result as, "wordpress woocommerce" whereas "go get math" returns no results.  "wordpress plugin woo commerce" works as a query using the code I wrote currently at this time.